### PR TITLE
Fix some grammar/punctuation + rewrite stuff

### PR DESCRIPTION
**Pull request by phonetic112**
Some general grammar + punctuation cleanup, consolidation, rephrasing, etc.

### CODE REVIEW

File: pages/Configuring/Window-Rules.md [hunk 1]
```diff
@@ -209,26 +210,26 @@ windowrulev2 = opacity 0.8 0.8,class:^(kitty)$
...
--> all kitty windows will have opacity 0.8, also if they are floating. -> all
-other floating windows will have opacity 0.5.
+Here, all kitty windows will have `opacity 0.8`, even if they are floating. 
+The rest of the floating windows will have `opacity 0.5`.
 
 {{< callout type=info >}}
 
-Opacity is a PRODUCT of all opacities by default. E.g. `active_opacity` to 0.5 and
-windowrule opacity to 0.5 will result in a total opacity 0.25. You are allowed
+Opacity is a PRODUCT of all opacities by default. Setting `activeopacity` to 0.5
```
Comment (fufexan) [inline pages/Configuring/Window-Rules.md:226]:
```suggestion
Opacity is a PRODUCT of all opacities by default. For example, setting `activeopacity` to 0.5 
```

File: pages/Getting Started/Installation.md [hunk 2]
```diff
@@ -19,14 +19,14 @@ might have **major** issues running Hyprland.
 
 ## Installation
 
-Installing Hyprland is very easy. Either you install it from your local package
-provider (if they provide pkgs for Hyprland) or you install/build it yourself.
+Installing Hyprland is very easy. Simply install it with your package manager 
+(if the pkg is available for Hyprland) or install/build it yourself.
```
Comment (fufexan) [inline pages/Getting Started/Installation.md:23]:
```suggestion
(if there is a Hyprland package available) or install/build it yourself.
```

File: pages/Useful Utilities/xdg-desktop-portal-hyprland.md [hunk 3]
```diff
@@ -167,5 +167,5 @@ logs.
 
 `systemctl --user status xdg-desktop-portal-hyprland`
 
-if you see a crash, it's most likely you are missing `qt6-wayland` and/or
+If you see a crash, it's likely you are missing either `qt6-wayland` and/or
```
Comment (fufexan) [inline pages/Useful Utilities/xdg-desktop-portal-hyprland.md:170]:
```suggestion
If you see a crash, it's likely you are missing either `qt6-wayland` or
```

**Review by fufexan — CHANGES_REQUESTED** [review]

**Comment by phonetic112** [comment]
should be good now

**Review by fufexan — APPROVED** [review]
LGTM! Thanks for the hard work!

**Pull request merged**